cupy/core/core.pyx

```diff
@@ -1840,9 +1840,10 @@ _HANDLED_TYPES = (ndarray, numpy.ndarray)
 # TODO(niboshi): Move it out of core.pyx
 
 cdef list _cupy_header_list = [
-    'cupy/complex.cuh',
-    'cupy/carray.cuh',
     'cupy/atomics.cuh',
+    'cupy/carray.cuh',
+    'cupy/complex.cuh',
+    'cupy/cuComplex_bridge.h',
```

Comment:
I think internally we almost always prepend the headers, but I suppose having `cuComplex_bridge.h` included is harmless to all internal usages.